api/dss.yaml

```diff
@@ -998,36 +1074,33 @@ components:
       type: object
       properties:
         distance:
-          format: float
           description: Distance above reference datum.  This value is provided in
-            meters and must have a minimum resolution of 1 meter.
-          type: number
+            millimeters and must have a minimum resolution of 1 meter.
+          type: integer
         reference:
           description: The reference datum above which the height is reported.
           enum:
           - TakeoffLocation
           - GroundLevel
           type: string
     Latitude:
-      format: double
       description: Degrees of latitude north of the equator, with reference to the
-        WGS84 ellipsoid.
-      maximum: 90
+        WGS84 ellipsoid, represented as lat * 1e7.
+      maximum: 900000000
       exclusiveMaximum: false
-      minimum: -90
+      minimum: -900000000
       exclusiveMinimum: false
-      type: number
-      example: 34.123
+      type: integer
```

Comment:
In the absence of a very compelling problem that would be caused by not changing all these doubles to integers, we should not change the API this much.

Reply:
reverting changes here for now, but let's circle back in the future